DEPLOY.md

```diff
@@ -130,6 +130,51 @@ Note that the angle brackets [are only required if there are spaces in the URL](
 
 In effect, this means that linking to a section is as simple as copy-pasting its name in the URL field, prepending a `#`, and wrapping everything in `<>` if the name contains a space.
 
+### Bit breakdown tables
```

Comment:
Are we sure this goes in DEPLOY.md and not CONTRIBUTING?

Reply:
I agree, but all of the custom syntax is documented here: better keep it all together now, and move it in a separate PR?

Reply:
Agree!